code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -751,6 +751,18 @@ parameter_types! {
 	  pub MinReward: Balance = 10 * CurrencyId::unit::<Balance>();
 	  pub Stake: Balance = 10 * CurrencyId::unit::<Balance>();
 }
+parameter_types! {
+		pub const PalletXcmIbcInstanceId: u8 = 192; // PalletXcmIbc: pallet_xcm_ibc = 192,
```

Comment:
you can get id from pallet. see simulator tests.

Reply:
no need copy paste.

Reply:
can you please point the place where i can use the pallet to pallet id?

Reply:
```
pub fn move_runtime_pallet<
	const OLD_NAME: &'static str,
	const MIGRATED_STORAGE_VERSION: u16,
	NewPallet: PalletInfoAccess + GetStorageVersion,
>() -> Weight {
	let new_pallet_name = <NewPallet as PalletInfoAccess>::name();
```

Reply:
```
fn fun(){
	let _ = <Ibc as PalletInfo>::index().unwrap();
}
```

Reply:
Thank you so much.